treeherder/model/management/commands/import_perf_data.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.core.management.base import BaseCommand, CommandError
+from optparse import make_option
+from treeherder.client import PerfherderClient
+from treeherder.model.derived.jobs import JobsModel
+
+import concurrent.futures
```

Comment:
> maybe something else is already pulling it in.

peep passes --no-deps to pip, so all dependencies have to be specified explicitly.

It's not present in my vagrant environment:
```
(venv)vagrant@local:~$ python
Python 2.7.9 (default, Apr 15 2015, 20:04:59)
[GCC 4.6.3] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import concurrent.futures
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named concurrent.futures
```